vizro-core/src/vizro/models/_components/table.py

```diff
@@ -51,12 +53,35 @@ def __getitem__(self, arg_name: str):
             return self.type
         return self.figure[arg_name]
 
+    def _build_underlying_table_object(self):
+        data = data_manager._get_component_data(self.id)  # type: ignore
+        kwargs = self.figure._arguments.copy()
+        kwargs.pop("data_frame", None)
```

Comment:
Do we still have to do the popping of data_frame? I slightly remember we removed this from Max PR? Or am I remembering this incorrectly?

Reply:
I change it now.